tcf_website/api/views.py

```diff
@@ -125,9 +125,10 @@ def get_queryset(self):
         # TODO: Refactor using django-filter if possible
         # Can't use `.filter()` twice, so use a dict
         # https://stackoverflow.com/q/8164675/
-        params = {}
+        params = {'year__gte': Semester.latest().year - 5}
         if 'course' in self.request.query_params:
             params['section__course'] = self.request.query_params['course']
         if 'instructor' in self.request.query_params:
             params['section__instructors'] = self.request.query_params['instructor']

```

Comment:
The double asterisks in the next line (`return self.queryset.filter(**params)...`) is called dictionary unpacking in Python. For example, if the conditions in the two if statements are both true, `params` will evaluate to the following dictionary:
```
params = {
    'year__gte': Semester.latest().year - 5,
    'section__course': self.request.query_params['course'],
    'section__instructors': self.request.query_params['instructor'],
}
```
And after dictionary unpacking, it will be equivalent to
```
return self.queryset.filter(
    year__gte=Semester.latest().year - 5,
    section__course=self.request.query_params['course'],
    section__instructors=self.request.query_params['instructor'],
)...
```